src/components/Api.js

```diff
@@ -0,0 +1,96 @@
+export default class Api { 
+  constructor({ baseUrl, headers }) {
+      this._baseUrl = baseUrl;
+      this._headers = headers;
+  }
+
+  _checResponse(res) {
+      if (res.ok) {
+          return res.json();
+      }
+      return Promise.reject(`Error: ${res.status}`);
+  }
+
+  _request(name, link) {
+      return fetch(name, link).then(this._checResponse);
+  }
+
+  getUserInfo(){
+      return this._request(`${this._baseUrl}/users/me`, {
+        headers: this._headers,
+      }).finally(() => {

```

Comment:
You don't need `finally` with just `console.log`. Please, remove them to clean up the code